stdlib/argparse.pyi

```diff
@@ -395,7 +395,7 @@ if sys.version_info >= (3, 12):
                 deprecated: bool = False,
             ) -> None: ...
             @overload
-            @deprecated("The `type`, `choices`, and `metavar` parameters are ignored and will be removed in Python 3.14.")
+            @deprecated("The `type`, `choices`, and `metavar` parameters are ignored and removed in Python 3.14.")
```

Comment:
The old wording suggested that they were ignored on all versions, but they will be removed on Python 3.14. The new wording suggests that they are only ignored on 3.14, and they're also removed on 3.14, but that doesn't make much sense, I don't think

Reply:
Thanks, I have problems with English :)